docs/conflicts.md

```diff
@@ -85,22 +85,30 @@ you, making it easier to spot the differences to apply to the other side. Here's
 how that would look for the same example as above:
 
 ```
-  <<<<<<<
-  %%%%%%%
+  <<<<<<< Conflict 1 of 1
+  %%%%%%% Changes from base to side #1
    apple
   -grape
   +grapefruit
    orange
-  +++++++
+  +++++++ Contents of side #2
   APPLE
   GRAPE
   ORANGE
-  >>>>>>>
+  >>>>>>> Conflict 1 of 1 ends
 ```
 
 As in Git, the `<<<<<<<` and `>>>>>>>` lines mark the start and end of the
-conflict. The `%%%%%%%` line indicates the start of a diff. The `+++++++`
-line indicates the start of a snapshot (not a diff).
+conflict. The `%%%%%%%` line indicates the start of a diff. The `+++++++` line
+indicates the start of a snapshot (not a diff). The text after the conflict
+markers is merely a guide for the human reader.
+
+The base of the conflict corresponds to the base of Git's conflict, and the two
+sides correspond to the left/right sides of Git's conflict in some order.
```

Comment:
The drawing of similarities with Git should have ended before this point. Suffice to say here that the assignment of the sides to `#1` and `#2` convey no intended meaning.